### PR TITLE
readme and istio changes + lots of static assets

### PR DESCRIPTION
**Pull request by dshimo**

### CODE REVIEW

File: docs/istio.md [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+    + [Access Grafana Dashboard](#access-grafana-dashboard)
+    + [Access Service Graph Dashboard](#access-service-graph-dashboard)
+    + [Access Jaeger Tracing Dashboard](#access-jaeger-tracing-dashboard)
+    + [Access Kiali Dashboard](#access-kiali-dashboard)
+  * [Cleanup](#cleanup)
+  * [Conclusion](#conclusion)
+
+## Introduction
+
+The journey to cloud-native microservices comes with great technical benefits. As we saw in the microservices reference architecture (Bluecompute) we were able to individually deploy, update, test, and manage individual microservices that comprise the overall application. By leveraging `Helm`, we are able to individually package these services into charts and package those into an umbrella chart that deploys the entire application stack conveniently and quickly.
```
Comment (kevinwlau) [inline docs/istio.md]: Remove both instances of "individually"

File: docs/istio.md [hunk 2]
```diff
@@ -0,0 +1,631 @@
...
+
+Having such flexibility comes at a price though. For example, the more microservices you have, the more complicated it becomes to manage, deploy, update, monitor, and debug. Thankfully, the Kubernetes community acknowledge these limitations and has provided us with the concept of a `Service Mesh`. As explained [here](https://istio.io/docs/concepts/what-is-istio/#what-is-a-service-mesh), the term "service mesh" describes the network of microservices that make up applications and the interactions between them. Examples of service mesh projects include [OpenShift](https://www.openshift.com/), developed by RedHat, and [`Istio`](https://istio.io/), co-developed by IBM and Google. Featured in Bluecompute, Istio aims to help you connect, secure, control, and observe your services in a standardized and language-agnostic way that doesn't require any code changes to the services.
```
Comment (kevinwlau) [inline docs/istio.md]: I don't think Service Mesh and Istio should be put into a codeblock.

File: docs/istio.md [hunk 3]
```diff
@@ -0,0 +1,631 @@
...
+* [helm](https://github.com/kubernetes/helm) (Kubernetes package manager) - Follow the instructions [here](https://github.com/kubernetes/helm/blob/master/docs/install.md) to install it on your platform.
+	+ If using `IBM Cloud Private`, we recommend you follow these [instructions](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.0/app_center/create_helm_cli.html) to install `helm`.
+	+ If using IBM Cloud Kubernetes Service (IKS), please use the most up-to-date version of helm
+
+## Blue-Compute Istiofied
+
+As with any complex application architecture, we had to make some changes to fully support the `bluecompute-ce` application in the Istio service mesh. Luckily, those changes were minimal but were necessary to leverage most of Istio's features and follow best practices.
+
+### Architecture
+![Architecture](../static/images/diagram_bluecompute_openshift.png)
```
Comment (kevinwlau) [inline docs/istio.md]: This diagram has OpenShift handling the routing

File: docs/istio.md [hunk 4]
```diff
@@ -0,0 +1,631 @@
...
+
+You will notice that, compared to the original [architecture diagram](../static/images/app_architecture.png), the overall application remains the same, minus a few additions:
```
Comment (kevinwlau) [inline docs/istio.md]: Should this be the diagram shown on the page instead? And move the OpenShift diagram into another section

File: docs/istio.md (hunk 2 above)
Comment (kevinwlau) [reply docs/istio.md]: Same for Helm in the paragraph above

File: docs/istio.md [hunk 5]
```diff
@@ -2,188 +2,162 @@
...
-  + The exception are the [StatefulSet-Based Services](#statefulset-based-services) (Elasticsearch, CouchDB, and MariaDB), which don't work well with Istio at the moment.
-  + All details explained in the [StatefulSet-Based Services](#statefulset-based-services) section.
-  + The Envoy proxy can also be used to establish Mutual TLS connections between the services if desired, which we will explain in later sections.
-  + The Envoy proxy is also used to collect rich metrics, which will explore in later sections.
+![Architecture](../static/images/diagram_bluecompute_istio.png)
 
-A majority of the application was successfully modified to work with and leverage Istio's features. For the services that couldn't be made to work with Istio, we were still able to expose them to the service mesh and establish connections with those services in the service mesh. More details on that in later session.
+You will notice that, compared to the original [architecture diagram](../static/images/app_architecture.png), the
+overall application remains the same, minus a few additions:
```
Comment (kevinwlau) [inline docs/istio.md]: with* a few additions

File: docs/istio.md [hunk 6]
```diff
@@ -481,71 +517,109 @@ Now, open a new browser tab and go to http://localhost:3000 to open Grafana's ho
...
 ![Architecture](../static/images/grafana_2_dashboard_select.png)
 
-On the service drop-down at the top left, select `web.default.svc.cluster.local` service. You should be presented with a dashboard similar to the following:
+On the service drop-down at the top left, select `web.default.svc.cluster.local` service. You should be presented with a
+dashboard similar to the following:
 
 ![Architecture](../static/images/grafana_3_web_dashboard.png)
 
-I'll leave it up to you to examine all the dashboards. The main thing I want to point out is the `Incoming Requests by Source And Response Code` dashboard at the bottom left. The load you see is generated from our script. If you were to interrupt the script, you will see, upon refreshing, that the load will go down to 0. Also notice that the source for all of these requests come directly from the `istio-ingressgateway.istio-system` since we are running our load generation script against the gateway itself instead of through a port-forwarding connection.
+I'll leave it up to you to examine all the dashboards. The main thing I want to point out is the
```
Comment (kevinwlau) [inline docs/istio.md]: Possibly a future change: we might want to rewrite to not use first person.
No change requested for now

**Review by evie-lau — CHANGES_REQUESTED** [review]
Other changes requested in-person

**Review by evie-lau — APPROVED** [review]

**Pull request merged**